go/vt/vtgate/engine/insert_test.go

```diff
@@ -1192,10 +1461,10 @@ func TestInsertShardedIgnoreUnownedVerify(t *testing.T) {
 		`ExecuteMultiShard ` +
 			`sharded.20-: prefix mid1 suffix /* vtgate:: keyspace_id:166b40b44aba4bd6 */ ` +
 			`{_c30: type:INT64 value:"10" _c32: type:INT64 value:"12" ` +
-			`_id0: type:INT64 value:"1" _id1: type:INT64 value:"2" _id2: type:INT64 value:"3" } ` +
```

Comment:
Why did these expected results change?

Reply:
Reposting from a different conversation: "It was an optimization that happened as a result of the refactor. Previously, the bind vars were computed at different stages of processing. so, if some rows were later removed, those bind vars stayed. In the new case, the bind vars are computed after all rows are removed."